cmds/webboot/network.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	ui "github.com/gizak/termui/v3"
+	"github.com/u-root/NiChrome/pkg/wifi"
+	"github.com/u-root/webboot/pkg/menu"
+	"github.com/vishvananda/netlink"
+)
+
+func connected() bool {
```

Comment:
Just in my opinion, I think you can have a new pkg and move functions that mostly focus on network things to there. But remain functions that mostly focus on UI there. Like he connected(), interfaceIsWireless() .etc could be moved to the new pkg

Reply:
I'm not sure it's worth making a separate package yet. `connected()` and `interfaceIsWireless()` total to about 10 lines, and then the rest of the code involves the menu flow from the main program.